lib/scrabble/player.rb

```diff
@@ -1,11 +1,14 @@
-require "scrabble.rb"
+require "./lib/scrabble_module"
 module Scrabble
   class Player
     attr_reader :name, :total_score
+    attr_accessor :tiles
+
     def initialize(name)
       @name = name
       @plays = []
       @total_score = 0
+      @tiles = []
     end
 
     def plays
```

Comment:
I think it would make more sense to have an `attr_accessor` for this rather than having a separate method